src/display/editor/tools.js

```diff
@@ -472,6 +585,19 @@ class AnnotationEditorUIManager {
     this.#commandManager.destroy();
   }
 
+  get hcmFilter() {
+    return shadow(
+      this,
+      "hcmFilter",
+      this.#filterFactory && this.#pageColors
```

Comment:
It shouldn't be possible for the `filterFactory` to be undefined here, since we're getting that from the `pdfDocument`-instance in the constructor now.
```suggestion
      this.#pageColors
```